parsers/lib/quality.py

```diff
@@ -201,4 +201,13 @@ def validate_production(obj: Dict[str, Any], zone_key: ZoneKey) -> None:
                 % (key, zone_key)
             )
 
+    if all(
+        (obj.get("production").get(key) == 0)
+        or (obj.get("production").get(key) is None)
+        for key in obj.get("production", {})
```

Comment:
```suggestion
       (value == 0)
        or (value is None)
        for value in obj.get("production", {}).values()
```
Makes it a bit easier to read.

Reply:
Maybe do not commit it like this.

Reply:
As discussed, please just add the check for 'storage only zone'